src/arithmetic.jl

```diff
@@ -33,8 +33,8 @@ Reverse multiplication
 """
 function mul_rev(a::Interval, b::Interval, c::Interval)  # a = b * c
 
-    b = b ∩ (a / c)
-    c = c ∩ (a / b)
+    ((0.0 ∉ a) ||  (0.0 ∉ b)) && (c = c ∩ (a / b))
```

Comment:
Extra space after the `||`.

Reply:
I would prefer `( (0 ∉ a) || (0 ∉ b) )` with spacing around the outside parentheses.